src/firefly/js/ui/FitsDownloadDialog.jsx

```diff
@@ -147,19 +127,51 @@ function getInitState()  {
 }
 
 const FitsDownloadDialogForm= memo( ({isWs, popupId, groupKey}) => {
-    const [{pv,plot,hasOperation,threeC,colors}]= useState(getInitState);
-    useBindFieldGroupToStore(groupKey); // this will force a rerender when anything changes
-    const band = threeC ? getFieldVal(groupKey, 'threeBandColor', colors[0]) : Band.NO_BAND.key;
-    const totalChildren = (isWs ? 3 : 2) +  (hasOperation ? 1 : 0) + (threeC ? 1 : 0);// fileType + save as + (fileLocation)
-    const childH = (totalChildren*(20+mTOP));
+    const [{pv, plot, hasOperation, threeC, colors}] = useState(getInitState);
```

Comment:
I took one more look at this line and realized that is it completely unnecessary and wrong. We are doing a `useState` without a set. That means that it does not function as state at all. All these fields could be computed in line. 

Remove `getInitState` and replace with.

```js
const pv= getActivePlotView(visRoot());
const plot = primePlot(pv);
const colors= getColors(plot);
const threeC= isThreeColor(plot);
const hasOperation= plot?,plotState.hasOperation(Operation.CROP) ?? false;
```

If we wanted to dialog to update when the user switches plots (under the dialog) we would use `useStoreConnector` and the replace the `const pv=...` with
```js
 const pv= useStoreConnector( () => getActivePlotView(visRoot()));
```
but that is not how it works right now (maybe if we ever get that request we will do that.)

I think I put this line in when I converted it to a functional component, I should have recognized the problem then.